### ISSUE TITLE
Provide EoL / Support information on the website

### ISSUE THREAD

**Issue by captn3m0** [roundcube/roundcube.github.com#44]
In context to the PR at https://github.com/endoflife-date/endoflife.date/pull/336 - the Roundcube website doesn't seem to have any of this information available.

1. Does the RoundCube project have a documented support policy? Something that states how long minor releases get bug fixes.

2. Is there a separate "active support" (bug fixes/features) v/s "critical support" (security or critical bug fixes only) for the releases?

3. Is there a "expected" release schedule?

Basically looking for Roundcube's version of this page: https://www.php.net/supported-versions.php

Happy to file a PR here as well, if a maintainer can confirm the data at https://deploy-preview-336--endoflife-date.netlify.app/roundcube

**Comment by alecpl** [comment]
The answer to all questions is "No".
